src/components/citizen-list-item/index.test.tsx

```diff
@@ -19,19 +19,25 @@ describe(CitizenListItem, () => {
   it('renders the user display name', function () {
     const wrapper = subject({ user: { firstName: 'John', lastName: 'Doe' } as any });
 
-    expect(wrapper.find(c('name')).text()).toEqual('John Doe');
+    const name = wrapper.find(c('name'));
+
+    expect(name).toHaveText('John Doe');
```

Comment:
My personal preference would be to inline this variable. "Finding the node" isn't really part of the test so having it as a separate step is slightly confusing. Not worth changing just pointing out my thoughts.

```
expect(wrapper.find(c('name'))).toHaveText('John Doe');
```